.github/workflows/github_release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+base_dir="$( cd "$(dirname "$0")/../.." >/dev/null 2>&1 ; pwd -P )"
+readonly base_dir
+readonly pom_file="$base_dir/parent/pom.xml"
+
+# Read project version from pom file
+project_version=$(grep "<revision>" "$pom_file" | sed --regexp-extended 's/<.{0,1}revision>//g' | xargs)
```

Comment:
```suggestion
project_version=$(grep "<revision>" "$pom_file" | sed --regexp-extended 's/<.{0,1}revision>//g')
```

The `xargs` looks superfluous in this scenario.

Reply:
It was required to trim whitespace. I updated the regex to avoid this.